src/client/app/frameworks/i18n/components/lang-switcher.component.ts

```diff
@@ -3,11 +3,12 @@ import {Store} from '@ngrx/store';
 import 'rxjs/add/operator/take';
 
 // app
-import {FormComponent, Config, LogService, ILang} from '../../core/index';
+import {Config, LogService, ILang} from '../../core/index';
 import {ElectronEventService} from '../../electron/index';
 import {MultilingualService} from '../index';
+import {Component} from '@angular/core';
 
-@FormComponent({
+@Component({
```

Comment:
Not using the decorator here will of course make this component unavailable to the {N} app... just fyi.